ert3/engine/_run.py

```diff
@@ -150,14 +150,17 @@ def _load_ensemble_parameters(
 
 def _prepare_sensitivity(
     ensemble: ert3.config.EnsembleConfig,
+    experiment_config: ert3.config.ExperimentConfig,
     parameters_config: ert3.config.ParametersConfig,
     workspace_root: pathlib.Path,
     experiment_name: str,
 ) -> None:
     parameter_distributions = _load_ensemble_parameters(
         ensemble, parameters_config, workspace_root
     )
-    input_records = ert3.algorithms.one_at_the_time(parameter_distributions)
+    input_records = ert3.algorithms.one_at_the_time(
+        parameter_distributions, experiment_config.tail
```

Comment:
It is good practise to pass keyword arguments with name, like: `tail=experiment_config.tail`